docs/glossary.md

```diff
@@ -0,0 +1,25 @@
+# Table of Contents
+
+# Dependency topics
+
+- Direct
+- Transitive
```

Comment:
We should use the Deep term for consistency with UI

Reply:
Maybe - I'm considering shifting the UI to use "transitive" instead. "Transitive" is more right, although "deep" tends to be more intuitive for non-technical users.

Reply:
Yea I would agree that "deep" is more intuitive for non-technical users.